src/parser/DASHTree.cpp

```diff
@@ -719,24 +727,6 @@ start(void *data, const char *el, const char **attr)
           dash->strXMLText_.clear();
           dash->currentNode_ |= MPDNODE_BASEURL;
         }
-        else if (strcmp(el, "Role") == 0)
```

Comment:
What is the reason removing the Role element?

Reply:
it is double, look line 584

Reply:
Yes, thx for finding / contributing